test/spec/runners/epoch_processing.ex

```diff
@@ -0,0 +1,61 @@
+defmodule EpochProcessingRunner do
```

Comment:
To be consistent, this should end with `TestRunner`:

```suggestion
defmodule EpochProcessingTestRunner do
```